src/fe-common/core/hilight-text.h

```diff
@@ -7,7 +7,7 @@
 
 #include "formats.h"
 
-typedef struct _HILIGHT_REC HILIGHT_REC;
+//typedef struct _HILIGHT_REC HILIGHT_REC;
```

Comment:
typedef struct _HILIGHT_REC HILIGHT_REC is defined in formats.h so that HILIGHT_REC can be used in _TEXT_DEST_REC, maybe the author of the patch should've removed that line all together but all I did was lift the patch into github and create a PR for it :)